Repurpose the manifesto-app folder and add draft Kyma Elevator Pitch and Personas (#474)
kyma-project/community: manifesto/personas.md

# Kyma Personas

> The following personas are fictional characters, which we defined in order to represent the different user types that might use Kyma. Keep in mind those personas when contributing to Kyma as they will help you to understand Kyma users' needs and goals (GAINS :green_heart:) and major pain points (PAINS :broken_heart:).

Reviewer: ```suggestion
The following personas are fictional characters that we defined in order to represent the different types of users that might use Kyma. Keep these personas in mind when contributing to Kyma as they will help you to understand Kyma users' needs and goals (GAINS :green_heart:) and major pain points (PAINS :broken_heart:).
```

Reviewer: The construction is `keep sb/sb in mind`. See the [reference](https://idioms.thefreedictionary.com/keep+in+mind).

1. [Matthew](#matthew---cloud-native-software-engineer)
2. [Nicolas](#nicolas---tech-lead)

> ## Matthew - Cloud Native Software Engineer

Reviewer: ```suggestion
## Matthew - Cloud-Native Software Engineer
```

Author: As discussed, let's go with `Cloud Native` for now.

> ![Mathew](assets/matthew.png)

Reviewer 2: done

|                             |                                                                                                 |
| --------------------------- | ----------------------------------------------------------------------------------------------- |
| Name                        | **Matthew**                                                                                     |
| Age                         | 26                                                                                              |

> | Ocupation                   | Cloud Native Engineer at **PureJoy**                                                            |

Reviewer: ```suggestion
| Occupation                   | Cloud-Native Engineer at **PureJoy**                                                            |
```

> | Characteristic              | Open source ethusiast and contributor. You will meet him at the webinars and community meetups. |

Reviewer: ```suggestion
| Characteristics              | Open source enthusiast and contributor. You will meet him at webinars and community meetups. |
```

> | K8S experience (1-6)  | :book: :book: :book: :book: :book: :book:                                                       |

Reviewer: ```suggestion
| Kubernetes experience (1-6)  | :book: :book: :book: :book: :book: :book:                                                       |
```

> | K8S experience              | :book: :book: :book: :book: :book: :book:                                                       |
> | Accountability/Decisiveness | :sweat: :sweat:                                                                                 |

Reviewer 3: What is the rage for `K8S experience` and `Accountability/Decisiveness`? I was wondering whether it's (1-6) or even (1-10)
Could this range be added to the first column?

Could also 💪  be used instead of 😓  for grading `Accountability/Decisiveness` ?
```suggestion
| K8S experience (1-6)  | :book: :book: :book: :book: :book: :book:                                                       |
| Accountability/Decisiveness (1-6) | 💪  💪                                                                                  |
```

> | Slogans                     | _Is there a CLI for that?_, _Can I automate it?_                                                |

Reviewer: ```suggestion
| Slogans                     | _Is there a CLI for that?_
 _Can I automate it?_                                                |
```

Reviewer: Instead of a comma, I'd move the second slogan to the next line.

Author: Just in case you need it in the future: to do what you have in mind, you need to use `<br>` instead of plain `Enter/Return`.
```suggestion
| Slogans                     | _Is there a CLI for that?_ <br> _Can I automate it?_                                                |
```

### GAINS

> - :green_heart: I want to write my K8S application using my favorite IDE and store it in github.

Reviewer: ```suggestion
- :green_heart: I want to write my Kubernetes application using my favorite IDE and store it in GitHub.
```

> - :green_heart: I want to build K8S applications in any language, really fast.

Reviewer: ```suggestion
- :green_heart: I want to build Kubernetes applications in any language, really fast.
```

- :green_heart: I need a thin "multi-cloud" runtime for my application.
- :green_heart: I need a super easy, automated deployment directly from my application's source code.

> - :green_heart: Help me understand Kyma API

Reviewer: ```suggestion
- :green_heart: Help me understand Kyma API.
```

### PAINS

> - :broken_heart: I hate doing backward engineering in order to understand how thigs work together

Reviewer: ```suggestion
- :broken_heart: I hate doing backward engineering in order to understand how things work together.
```

> - :broken_heart: Dont make me use code editors in the web.

Reviewer: ```suggestion
- :broken_heart: Don't make me use code editors on the web.
```

> - :broken_heart: I dont want to be locked with any tools. Let me replace or completely opt out from any of the pre-configured components

Reviewer: ```suggestion
- :broken_heart: I don't want to be locked with any tools. Let me replace or completely opt out of any of the preconfigured components.
```

> - :broken_heart: I hate tons of configuration

Reviewer: ```suggestion
- :broken_heart: I hate tons of configuration.
```

## Nicolas - Tech Lead

![Nicolas](assets/nicolas.png)

|                             |                                                                                                                                                                                                         |
| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Name                        | **Nicolas**                                                                                                                                                                                             |
| Age                         | 36                                                                                                                                                                                                      |

> | Ocupation                   | Tech Lead (Technology evangelist) at **PureJoy**                                                                                                                                                        |

Reviewer: ```suggestion
| Occupation                   | Tech Lead (Technology evangelist) at **PureJoy**                                                                                                                                                        |
```

> | Characteristic              | Nicolas wants to bring innovation to products developed by **PureJoy**. He understands benefits of using opinionated tools from CNCF landscape. He needs to convince his boss and his development team. |

Reviewer: ```suggestion
| Characteristics              | Nicolas wants to bring innovation to products developed by **PureJoy**. He understands the benefits of using opinionated tools from the CNCF landscape. He needs to convince his boss and his development team. |
```

> | K8S experience (1-6)             | :book: :book: :book: :book:                                                                                                                                                                             |

Reviewer: ```suggestion
| Kubernetes experience (1-6)             | :book: :book: :book: :book:                                                                                                                                                                             |
```

> | K8S experience              | :book: :book: :book: :book:                                                                                                                                                                             |
> | Accountability/Decisiveness | :sweat: :sweat: :sweat: :sweat: :sweat: :sweat:                                                                                                                                                         |

Reviewer 3: Please see my previous comment
```suggestion
| K8S experience (1-6)             | :book: :book: :book: :book:                                                                                                                                                                             |
| Accountability/Decisiveness (1-6) | 💪 💪 💪 💪 💪 💪                                                                                                                                                    |
```

> | Slogans                     | _I know what I want.. not yet sure how to achieve it_                                                                                                                                                   |

Reviewer: ```suggestion
| Slogans                     | _I know what I want.. not yet sure how to achieve it._                                                                                                                                                   |
```

### GAINS

> - :green_heart: I need a comprehensive set of opinionated tools. (CNCF approved, recognized by community).

Reviewer: ```suggestion
- :green_heart: I need a comprehensive set of opinionated tools (CNCF approved, recognized by the community).
```

> - :green_heart: I need a comprehensive set of opinionated tools. (CNCF-approved, recognized by the community).

Reviewer: ```suggestion
- :green_heart: I need a comprehensive set of opinionated tools (CNCF-approved, recognized by the community).
```

> - :green_heart: I need certinity that Kyma uses industry standarts.

Reviewer: ```suggestion
- :green_heart: I need certainty that Kyma uses industry standards.
```

- :green_heart: I need a trial version of Kyma before I can recommend it.

> - :green_heart: I dont want to take extra care about security/monitoring/tracing...

Reviewer: ```suggestion
- :green_heart: I don't want to take extra care about security/monitoring/tracing...
```

> - :green_heart: I need my developers in my organisation to be focused on the solutions rather than infrastracture.

Reviewer: ```suggestion
- :green_heart: I need developers in my organization to be focused on the solutions rather than infrastracture.
```

### PAINS

- :broken_heart: I'm missing success stories, references.
- :broken_heart: I don't want my dev teams to struggle.

> - :broken_heart: I'm missing best pracices, recepies of how to put things together.

Reviewer: ```suggestion
- :broken_heart: I'm missing best practices, recipes of how to put things together.
```
